### DISCUSSION TITLE
Sort Selection command

### DISCUSSION THREAD

**Discussion by GBrachetta** [tailwindlabs/tailwindcss#12199] [Ideas]
The new option (available via command palette) `Tailwind CSS: Sort Selection` is a great feature, and it would really shine if it could be added to settings.json to run on save, for example here:

```json
{
  "editor.codeActionsOnSave": {
    // Option for the extension to sort classes on save
  }
}
```

Or

```json
{
  "tailwindCSS.lint.sortClassesOnSave": true
}
```

It would be great if you could consider adding this as an option.

**Comment by alex-krasikau** [comment]
Hey @GBrachetta!

This feature is already available using the [Prettier Tailwind plugin](https://github.com/tailwindlabs/prettier-plugin-tailwindcss).

The `Tailwind CSS: Sort Selection` feature most likely exists to handle non-standard cases, such as sorting classes in a variable string, which are not supported by the plugin by default.
```
const a = "flex flex-gap mt-4";
```
